source/material_model/interface.cc

```diff
@@ -255,6 +255,49 @@ namespace aspect
     {}
 
 
+    template <int dim>
+    MaterialModelInputs<dim>::MaterialModelInputs(const FEValuesBase<dim,dim> &fe,
+                                                  const Introspection<dim> &introspection,
+                                                  const LinearAlgebra::BlockVector &solution_vector)
+      :
+      position(fe.get_quadrature_points()),
+      temperature(fe.n_quadrature_points, numbers::signaling_nan<double>()),
+      pressure(fe.n_quadrature_points, numbers::signaling_nan<double>()),
+      pressure_gradient(fe.n_quadrature_points, numbers::signaling_nan<Tensor<1,dim> >()),
+      velocity(fe.n_quadrature_points, numbers::signaling_nan<Tensor<1,dim> >()),
+      composition(fe.n_quadrature_points, std::vector<double>(introspection.n_compositional_fields, numbers::signaling_nan<double>())),
+      strain_rate(fe.n_quadrature_points, numbers::signaling_nan<SymmetricTensor<2,dim> >()),
+      cell (NULL)
```

Comment:
The only code changes (as in not a comment change) are this one being discussed, and deleting the line discussed in Ian's next comment.